docs/data/sdks/ios-swift/migration.md

```diff
@@ -14,44 +14,34 @@ To migrate to `Amplitude-Swift`, update your dependencies and instrumentation.
 * `Amplitude-iOS`: Maintenance iOS SDK
 * `Amplitude-Swift`: New iOS SDK
 
-## Dependency
+## Dependencies
 
-For CocoaPods installation:
-=== "Amplitude-iOS"
-    Add dependency to Podfile.
-    ```
-    pod 'Amplitude', '~> 8.14'
-    ```
+### CocoaPods
 
-=== "Amplitude-Swift"
-    Add dependency to Podfile.
-    ```
-    pod 'AmplitudeSwift', '~> 0.3'
-    ```
+Add `AmplitudeSwift` dependency to `Podfile`.
 
-For Swift Package Manager installation:
+```diff
+- pod 'Amplitude', '~> 8.14'
++ pod 'AmplitudeSwift', '~> 0.4'
```

Comment:
Will version `1.0` be released?

Reply:
Yes, I am hoping to get DET and Experiment in before 1.0.